documentation/guides/breakpoints-migration.md

```diff
@@ -0,0 +1,792 @@
+## New breakpoints
+
+TODO: write about the 5 new breakpoints
+
+### Media queries variables
+
+We created some Sass variables to handle the new breakpoints and most of the migration you need to do is replacing old media-queries or mixins with them in your code.
+
+```scss
+$p-breakpoints-xs-up: '(min-width: 0em)';
+$p-breakpoints-xs-down: '(max-width: -0.003125em)';
+$p-breakpoints-xs-only: '(min-width: 0em) and (max-width: 30.621875em)';
+
+$p-breakpoints-sm-up: '(min-width: 30.625em)';
+$p-breakpoints-sm-down: '(max-width: 30.621875em)';
+$p-breakpoints-sm-only: '(min-width: 30.625em) and (max-width: 47.996875em)';
+
+$p-breakpoints-md-up: '(min-width: 48em)';
+$p-breakpoints-md-down: '(max-width: 47.996875em)';
+$p-breakpoints-md-only: '(min-width: 48em) and (max-width: 64.996875em)';
+
+$p-breakpoints-lg-up: '(min-width: 65em)';
+$p-breakpoints-lg-down: '(max-width: 64.996875em)';
+$p-breakpoints-lg-only: '(min-width: 65em) and (max-width: 89.996875em)';
+
+$p-breakpoints-xl-up: '(min-width: 90em)';
+$p-breakpoints-xl-down: '(max-width: 89.996875em)';
+$p-breakpoints-xl-only: '(min-width: 90em)';
+```
```

Comment:
I added this to be referenced in the examples of the mixins migration, wdyt?

Reply:
Yes! Love that you included these!